tests/mobly/controllers/android_device_lib/adb_test.py

```diff
@@ -702,7 +702,22 @@ def test_getprops_when_empty_string_always_returned(self, mock_sleep):
 
   def test_forward(self):
     with mock.patch.object(adb.AdbProxy, '_exec_cmd') as mock_exec_cmd:
-      adb.AdbProxy().forward(MOCK_SHELL_COMMAND)
+      adb.AdbProxy().forward(['tcp:12345', 'tcp:98765'])
+      mock_exec_cmd.assert_called_with(
+          ['adb', 'forward', 'tcp:12345', 'tcp:98765'],
+          shell=False,
+          timeout=None,
+          stderr=None,
+      )
+  def test_reverse(self):
```

Comment:
Please add a blank line before this test case, thanks!

Reply:
Done.